build.sbt

```diff
@@ -88,7 +90,7 @@ lazy val rdf = crossProject
   .settings(commonSettings: _*)
   .settings(
     name := "banana-rdf",
-    libraryDependencies += scalaz
+    libraryDependencies += "org.scalaz" %%% "scalaz-core" % scalazVersion
```

Comment:
why do you remove the `scalaz` `val` here, when you have it defined in  `project/Dependencies.scala`?

Reply:
see my comment below